### PR TITLE
Migrate multi-inclusive volumes how-to from Knowledge Base to product-docs

### PR DESCRIPTION
**Pull request by carolstran**
### Description

When working on the subscription page tooltips, the only guide we could link to related to the “Multi-inclusive volumes” feature was [one in our Knowledge Base](https://support.emnify.com/hc/en/community/posts/5287107334556-How-to-set-up-multi-inclusive-volumes-).

This guide has the outdated brand design and is generally something that is better suited for our product documentation, so this PR migrates/restructures the guide and verifies the the content is up-to-date ✨ 

> ✅   _All feedback from the [CSM team](https://github.com/emnify/product-docs/pull/100#issuecomment-1463619457) has been addressed and approved._

> ✅ _Functional review from Bas (Growth PM) done and feedback applied._

### Additional Context

Internal ticket 🎟️ [PGR-226](https://emnify.atlassian.net/browse/PGR-226) (parent story: [PGR-208](https://emnify.atlassian.net/browse/PGR-208))

[PGR-226]: https://emnify.atlassian.net/browse/PGR-226?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[PGR-208]: https://emnify.atlassian.net/browse/PGR-208?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/how-tos/multi-inclusive-volumes.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+/>
+
+In the preceding example, the two selected devices (**Walking Mobile** and **TravelSpot**) will now be assigned our **New coverage policy**.
+
+## Best practices
+
+When multiple inclusive volumes are configured on your account, you can update the rate assigned to one device within the month. 
+
+:::caution
+These changes have an impact on your monthly bill.
```
Comment (techwriter-dev) [inline docs/how-tos/multi-inclusive-volumes.md:111]:
```suggestion
These changes will have an impact on your monthly bill.
```
or

```
Could affect your monthly bill.
```

File: docs/how-tos/multi-inclusive-volumes.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+To change a coverage policy's tariff, follow these steps:
+
+1. Navigate to the [**Device Policies**](https://portal.emnify.com/device-policies) page.
+
+<img
+  src={require('./assets/portal-device-policies.png').default}
+  alt=""
+/>
+
+2. Scroll down to your **Coverage Policies**
```
Comment (techwriter-dev) [inline docs/how-tos/multi-inclusive-volumes.md:140]: Perhaps this is another opportunity to add additional internal anchors that reflect the structure beneath **Device Policies** which would eliminate these first 2 steps through the use of a single link:

- `#service-policies`
- `#coverage-policies`

device-policies-structure

**Review by techwriter-dev — CHANGES_REQUESTED** [review]
Added some suggestions.

File: docs/how-tos/multi-inclusive-volumes.md (hunk 2 above)
Comment (carolstran) [reply docs/how-tos/multi-inclusive-volumes.md:140]: Yes we have a ticket for this open in Growth squad! That said, we already know what the anchors will be so I can already add them as part of this task good catch 🙌🏼

**Comment by carolstran** [comment]
FYI @techwriter-dev: Received a bunch of feedback from Jean-Eudes (Director of Customer Success, [see Slack thread](https://emnifyteam.slack.com/archives/C27MUBN4S/p1678442224043269)) so I'll update with his feedback and make sure it's approved before I ping for a final code review 😇

**Comment by carolstran** [comment]
Alright finally ready for a re-review @techwriter-dev 😅 🚀

**Comment by techwriter-dev** [comment]
> @techwriter-dev If you have any tips for optimizing the images without messing up the quality, let me know 😅 I did my best but they're still big/don't look the best 🙃

@carolstran There is no perfect solution. Taking the following example, I recreated the image (showing different devices) and placed it under yours. The only difference is I set my browser to zoom in at 125% before taking the screenshot. If I can get it to fit, I sometimes zoom in to 150%. The goal is to have an image that comes as close to possible to the sweet spot of 915 pixels wide. That's the horizontal limit of Docusaurus for images on a standard monitor (1920 x 1080) in fullscreen mode. On a MacBook the default display is resampled to (1512 x 982).

Here's the **flip side**: when you use the **click-to-zoom-in feature** we've deployed, _your_ image will look much better than my zoomed-before-capture one. However, for whatever reason, on page load the resampling of much larger images like yours is more lossy that the smaller one I produced. 

img-zoom-comparison

**Review by techwriter-dev — APPROVED** [review]
I left some comments, but they're not blocking.

**Comment by carolstran** [comment]
@techwriter-dev Thanks for sharing more about the images! I managed to decrease the image size since writing that and it's not ideal but ok for now (IMO). 

Makes me think we should pull up the ticket we have about optimizing images up in priority though bc it could be worth finding a better, even automated process for this 🤔

**Comment by carolstran** [comment]
Note about this PR: Waiting on a final OK from Bas about how we mentioned the Pro/Plus plans. Once I get the ✅ from him, I'll merge 🥳

**Pull request merged**